<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegionalizationVectorWizardPageForm</class>
 <widget class="QWizardPage" name="RegionalizationVectorWizardPageForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="groupBox_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Output</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_22">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_21">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_10">
            <item row="0" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Repository:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Layer Name:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <layout class="QGridLayout" name="gridLayout_20">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_11">
              <item row="0" column="0">
               <widget class="QLineEdit" name="m_repositoryLineEdit">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QToolButton" name="m_targetFileToolButton">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>...</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QToolButton" name="m_targetDatasourceToolButton">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>DB</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <widget class="QLineEdit" name="m_newLayerNameLineEdit">
              <property name="readOnly">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
